### DISCUSSION TITLE
ROCm on Linux Using a 6800xt

### DISCUSSION THREAD

**Discussion by hamsterdog6** [ROCm/ROCm#3519] [Q&A] [answered]
My main goal is to train YOLO Pytorch weights using object detection. I want to make use of my GPU and the CPU is only usable on Windows. Will I have the same problem as I have on Windows where it only uses the CPU? Is there a particular version of ROCm that will work better with the 6800xt? I'll be on Ubuntu 24.04. Does that affect this at all or is there a distro that would be better for my goal? I've gone the Google Colab route a couple of times using the $10 plan and I end up using all my units pretty quickly, a lot sometimes because of experimenting. And yes, I know that would be better in general but I'm trying to save money.

**Comment by zichguan-amd** [comment, marked as answer]
Hi @hamsterdog6, you can follow the steps here to get the latest ROCm on Linux: https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/quick-start.html, and install PyTorch on bare metal: https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/3rd-party/pytorch-install.html#using-a-wheels-package, and make sure that you have access to the GPU from PyTorch following these steps here: https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/3rd-party/pytorch-install.html#testing-the-pytorch-installation. You should be good to start with the latest ROCm (6.2.4) on Ubuntu 24.04.
